pylintrc

```diff
@@ -335,19 +326,19 @@ max-args=20
 
 # Argument names that match this expression will be ignored. Default to name
 # with leading underscore
-ignored-argument-names=_.*
+ignored-argument-names=^_
 
 # Maximum number of locals for function / method body
-max-locals=20
+max-locals=100
```

Comment:
100 local variables looks huge (it actually exceeds the maximal number of statements). I hated this warning, but it still made me think whether there's something I can simplify.